ionoscloud/resource_server_test.go

```diff
@@ -769,3 +827,128 @@ resource ` + ServerResource + ` ` + ServerTestResource + ` {
 	  }
     }
 }`
+
+const testAccCheckDataSourceServerWithLabels = testAccCheckServerCreationWithLabels + `
+data ` + ServerResource + ` ` + ServerDataSourceById + ` {
+  datacenter_id = ` + DatacenterResource + `.` + DatacenterTestResource + `.id
+  id			= ` + ServerResource + `.` + ServerTestResource + `.id
+}
+`
+
+const (
+	testAccCheckServerUpdateLabels = `
+resource ` + DatacenterResource + ` ` + DatacenterTestResource + ` {
+	name       = "server-test"
+	location = "us/las"
+}
+resource "ionoscloud_ipblock" "webserver_ipblock" {
```

Comment:
we don't need this for the test. remove ipblock and other extraneous resources or blocks like firewall

Reply:
@cristiGuranIonos done in a587496c4cd2e7e39ddca5d28621da37a154f51d